README.md

```diff
@@ -121,7 +121,7 @@ Note that it may run on lower end equipment though good performance is not guara
 
 <div align="center">
 
-<a href="https://asciinema.org/a/141608" target="_blank"><img src="https://asciinema.org/a/141608.png" /></a>
+<a href="https://asciinema.org/a/352310" target="_blank"><img src="https://asciinema.org/a/352310.png" /></a>
```

Comment:
Nice work, but I suggest to choose another preview for this image.
You can select this preview in asciinema.org website

Reply:
Thank you.
But I'm not sure what do you mean <b>this</b>?
It seems your comment doesn't contain any link.

Reply:
I talk about preface frame of the video